firrtl/src/main/scala/firrtl/ir/Serializer.scala

```diff
@@ -566,7 +566,12 @@ object Serializer {
     val layers = if (circuit.layers.nonEmpty) {
       implicit val b = new StringBuilder
       def layerIt(layer: Layer)(implicit indent: Int): Unit = {
-        b ++= s"${NewLine}"; doIndent(); b ++= s"layer ${layer.name}, ${layer.convention} :"
+        b ++= s"${NewLine}"
+        doIndent()
+        b ++= s"layer ${layer.name}, ${layer.convention}"
+        for (d <- layer.outputDir)
+          b ++= ", \"" ++ d ++ "\"";
```

Comment:
```suggestion
        for (d <- layer.outputDir) {
          b ++= ", \"" ++ d ++ "\""
        }
```
No need for semicolon, and stylistically I always push for curly braces around mutation